keyboards/noxary/valhalla_v2/readme.md

```diff
@@ -0,0 +1,32 @@
+# Noxary Valhalla V2
+
+![valhalla](https://cdn.shopify.com/s/files/1/0071/6377/3043/products/BG_AngleL_a247224d-f9ca-4093-b0de-df776e34b02d_900x.jpg?v=1598742611)
```

Comment:
Please rehost this image on Imgur.

Reply:
Done in [8e33530](https://github.com/qmk/qmk_firmware/pull/22914/commits/8e33530073e80e8a724c49938bd5f78b44a0e09b)